rust/core-lib/src/config.rs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(clippy::redundant_clone)]
```

Comment:
I think we can get rid of this now?

Reply:
We have `domain.clone ()` on L469, L496 and `value.to_owned ()` on L760.

Change to `domain` & `value`.